### PR TITLE
Added PER mask

### PR DESCRIPTION
**Pull request by yc-adi**

### CODE REVIEW

**Review by EricB-ADI — APPROVED** [review]

File: Tools/Bluetooth/conn_sweep.py [hunk 1]
```diff
@@ -427,89 +320,115 @@ def run_script_reset_board(sh_file):
...
+                mini_RCDAT = mini_RCDAT_USB(Namespace(atten=set_val))
+            
+            sleep(0.1)
+
+            print("\nReset the packet stats.")
+            hciSlave.cmdFunc(Namespace(cmd="0102FF00"), timeout=10.0)
+            hciMaster.cmdFunc(Namespace(cmd="0102FF00"), timeout=10.0)
+
+            print("\nSlave listenFunc")
+            hciSlave.listenFunc(Namespace(time=1, stats="False"))
```
Comment (kevin-gillespie) [inline Tools/Bluetooth/conn_sweep.py:342]: I think we can remove these two listen calls. I'm pretty sure the intended effect is to clear out any pending events before checking the packet statistics.

Comment (kevin-gillespie) [reply Tools/Bluetooth/conn_sweep.py:342]: Since we're doing this on lines 351 and 352, I think these 2 are redundant.

**Review by kevin-gillespie — CHANGES_REQUESTED** [review]

File: Tools/Bluetooth/conn_sweep.py (hunk 1 above)
Comment (yc-adi) [reply Tools/Bluetooth/conn_sweep.py:342]: Okay, I'll remove them. Is there a easy way to get all the HCI commands used by CMW?

Comment (kevin-gillespie) [reply Tools/Bluetooth/conn_sweep.py]: Not that I can think of. We can use the TRACE output while it's being tested to see what commands are sent. It's something along the lines of these sequences. 

```
HCI Reset
TxTest
EndTest
HCI Reset

HCI Reset
RxTest
EndTest
HCI Reset
```

**Review by kevin-gillespie — APPROVED** [review]

**Review by EricB-ADI — APPROVED** [review]

**Pull request merged**